packages/app/src/components/MenuBar/MenuBar.js

```diff
@@ -1,11 +1,12 @@
-import { FileMenu } from '@dhis2/analytics'
+import { FileMenu, VIS_TYPE_ALL, VIS_TYPE_CHARTS } from '@dhis2/analytics'
```

Comment:
Maybe `VIS_TYPE_ALL` and `VIS_TYPE_CHARTS` should be renamed, as they're not "types" themselves but groups of types, e.g. `VIS_TYPE_GROUP_ALL` ?

Reply:
Good point, renamed.